pkgs/applications/misc/obinskit/default.nix

```diff
@@ -0,0 +1,58 @@
+{ lib
+, stdenv
+, fetchurl 
+, xlibs
+, libxkbcommon
+, libudev
+, gcc-unwrapped
+, electron_3
+}:
+
+stdenv.mkDerivation rec {
+  pname = "obinskit";
+  version = "1.1.1";
+
+  src = fetchurl {
+    url = "http://releases.obins.net/occ/linux/tar/ObinsKit_${version}_x64.tar.gz";
+    sha256 = "0052m4msslc4k9g3i5vl933cz5q2n1affxhnm433w4apajr8h28h";
+  };
+
+  unpackPhase = ''
+    tar -xzf $src
+    export src='ObinsKit_1.1.1_x64'
+  '';
```

Comment:
Why is this needed? I seems you want to make sourceRoot be `ObinsKit_1.1.1_x64` so maybe try
```
sourceRoot = "ObinsKit_${version}_x64";
```

and drop this.